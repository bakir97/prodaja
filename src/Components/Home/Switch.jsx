import React from "react";
import FormGroup from "@material-ui/core/FormGroup";
import FormControlLabel from "@material-ui/core/FormControlLabel";
import Switch from "@material-ui/core/Switch";
const SwitchComponent = ({ changeFiltersState, filters }) => {
  return (
    <FormGroup row>
      <FormControlLabel
        control={
          <Switch
            color="primary"
            checked={filters}
            onChange={() => changeFiltersState()}
            value="checkedA"
          />
        }
        label="Show filters"
      />
    </FormGroup>
  );
};

export default SwitchComponent;
