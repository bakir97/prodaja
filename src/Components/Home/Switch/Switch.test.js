import Switch from "./index";
describe("Switch", () => {
  const wrapper = shallow(<Switch />);
  it("should render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
