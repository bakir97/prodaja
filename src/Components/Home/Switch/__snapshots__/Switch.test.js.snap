// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Switch should render 1`] = `
<WithStyles(FormGroup)
  row={true}
>
  <WithStyles(FormControlLabel)
    control={
      <WithStyles(Switch)
        color="primary"
        id="switch"
        onChange={[Function]}
        value="checkedA"
      />
    }
    label="Show filters"
  />
</WithStyles(FormGroup)>
`;
